src/lint/rules/indentation.rs

```diff
@@ -1,12 +1,6 @@
 use std::convert::TryInto;
 
-use crate::analysis::parsing::{expression::{CastContent, FunctionCallContent, ParenExpressionContent},
-                               lexer::TokenKind,
-                               statement::{self, CompoundContent, DoContent, ForContent, ForeachContent,
-                                           IfContent, SwitchCase, SwitchContent, WhileContent},
-                               structure::{MethodContent, ObjectStatementsContent},
-                               tree::TreeElementTokenIterator,
-                               types::{BitfieldsContent, LayoutContent, StructTypeContent}};
+use crate::analysis::parsing::{expression::{CastContent, FunctionCallContent, ParenExpressionContent}, lexer::TokenKind, parser::Token, statement::{self, CompoundContent, DoContent, ForContent, ForeachContent, IfContent, StatementContent, SwitchCase, SwitchContent, WhileContent}, structure::{DMLObjectContent, MethodContent, ObjectStatementsContent}, tree::TreeElementTokenIterator, types::{BitfieldsContent, LayoutContent, StructTypeContent}};
```

Comment:
This change to the import indentation is a downgrade, and I am not sure what prompted it? IMO I think using multiple branches of sub-modules in an import a little awkward (so I'd prefer multiple lines all starting with "crate::analysis::parsing::") , but I'd accept this as it was pre-change.